docs/source/guides/libraries/swift_ios.rst

```diff
@@ -0,0 +1,282 @@
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_005.png
+
+Before we launch the application and test it we should deploy Iroha on our local machine and launch it.
+
+There is good news - steps 1-18 shouldn't be done manually every time - here is the script which does it automatically.
+
+The script for iOS client installation and setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+iroha_preparation_script.sh
+"""""""""""""""""""""""""""
+
+  .. code-block:: bash
+
+    #!/bin/bash
+
+    #download ios client and update dependencies
+    git clone https://github.com/soramitsu/iroha-ios.git
+    cd iroha-ios/
+    carthage update --platform iOS
+    cd SwiftyIrohaExample
+    carthage update --platform iOS
+
+    #build grpc client for sample application
+    cd grpc-swift/
+    rm -rf ./*
+    rm -rf ./.*
+    git clone --branch 0.3.3 https://github.com/grpc/grpc-swift.git .
+    make
+
+    #back to the root where script was executed
+    cd ../../..
+
+    #download and build Iroha library for iOS
+    curl https://raw.githubusercontent.com/hyperledger/iroha/master/shared_model/packages/ios/ios-build.sh > ios-build.sh
+
+    #optional step - sometimes connection timeout appears when using git: scheme instead of https url
+    sed -i '' 's|git://github.com/hyperledger/iroha-ed25519|https://github.com/hyperledger/iroha-ed25519.git|g' ios-build.sh
+
+    #build library
+    chmod +x ios-build.sh
+    ./ios-build.sh SIMULATOR64 Debug
+
+    #place artefacts to proper sample's locations
+
+    # this command shows location for simulator artefacts
+    # use this command for device instead:
+    # mkdir -p iroha-ios/libs/iOS/
+    mkdir -p iroha-ios/libs/Simulator/
+
+    # this command shows location for simulator artefacts
+    # use this command for device instead:
+    # cp lib/* iroha-ios/libs/iOS/
+    cp lib/* iroha-ios/libs/Simulator/
+    cp -a include/. iroha-ios/headers/
+
+
+Starting Iroha Node
+^^^^^^^^^^^^^^^^^^^
+
+To run this example, you need an Iroha node up and running. Please check out
+:ref:`getting-started` if you want to learn how to start it.
+
+Launching Iroha iOS sample
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now it's time to switch back to SwiftyIrohaSample application and launch it on the simulator. Open Xcode project, select proper sample target and run.
+The sample will send test transaction to our node and query the result from blockchain. Successful operations will look similar to this Xcode console output:
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_007.png
+
+The output from Iroha terminal window (where the node is running):
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_008.png
+
+Great! We've sent our transaction to blockchain and verified that it's appeared.
```

Comment:
transaction <s>to blockchain</s> and verified <s>that it's appeared</s> **its presence in blockchain**